string name
VehicleStateCurv state_curv
VehicleStateGlob state_glob